ACPI/SSDT-BrcmFix.asl

```diff
@@ -0,0 +1,31 @@
+/**
+ * DW1820A Property Injection
+ */
+DefinitionBlock ("", "SSDT", 2, "OSY86 ", "BrcmFix", 0x00001000)
+{
+    External (DTGP, MethodObj)    // 5 Arguments
+    External (OSDW, MethodObj)    // 0 Arguments
+    External (\_SB.PCI0.RP03.PXSX, DeviceObj)
+
+    Scope (\_SB.PCI0.RP03.PXSX)
+    {
+        Method (_DSM, 4, NotSerialized)  // _DSM: Device-Specific Method
+        {
+            If (OSDW ())
+            {
+                If ((Arg0 == ToUUID ("a0b5b7c6-1318-441c-b0c9-fe695eaf949b")))
+                {
+                    Local0 = Package ()
+                        {
+                            "pci-aspm-default", Buffer() { 0x00 },
```

Comment:
Based on my test, you have to specify the Buffer size for this to be 0x02, 
i.e. `"pci-aspm-default", Buffer(0x02) { 0x00 },`, so the correct injected value  will be `data <00 00>`.
(The pci-aspm-default hex bits in MacOS can be 4 (e.g. 0x102, 0x103, 0x2, etc.))

Otherwise, the injected value length would be be wrong as `data <00>`, since by default the statement could be compiled as `"pci-aspm-default", Buffer(One) { 0x00 },`.
And this will cause kernel panic.

Reply:
That was in the sample SSDT in one of the links from your issue. I've changed it to 2 bytes, please try it now.